config/confluence/yaml/product-health-android.yaml

```diff
@@ -12,4 +12,13 @@ wiki_page:
           report-description: "Breakdown of bug status"
           attachment-filename: "bugs-caught-by-ui-tests-on-android.png"
           looker-graph-url: "https://mozilla.cloud.looker.com/looks/3041"
-          
+    - name: "Sentry"
+      reports:
+        - report-title: "Crash Free Users Rate"
+          report-description: "Crash Free Users Rate"
+          attachment-filename: "crash-free-rate-user.png"
```

Comment:
Where are the *.png files stored or generated? `api_confluence.py` seems to read from these yaml file and upload some image files (`image_attachments_*`).